tests/test-vip-plugins.php

```diff
@@ -153,12 +171,20 @@ public function test__modify_network_active_plugins() {
 		$this->assertTrue( $option_update );
 
 		/**
+		 * Check the raw value in the DB matches what we sent above - skips any filters
+		 */
+		$option_db = $wpdb->get_row( $wpdb->prepare( "SELECT meta_value FROM $wpdb->sitemeta WHERE meta_key = %s LIMIT 1", 'active_sitewide_plugins' ) );
```

Comment:
Minor: can simplify this using `get_var` which will return a string value instead of an object. Also, `wpdb->prepare` isn't strictly necessary here. We could just hard-code the meta_key inside the query.